### PR TITLE
PRO-6972: Remove Persistence

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-6972

### Change description ###
Remove Persistence service occurrences throughout the code and replace with Orchestrator where needed

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is the frontend application for the Probate Personal Applicants online service. The service provides a clear interface for citizens, presented as sequence of HTML 5 web pages designed to GDS Service Design guidelines, so that they can apply for Probate online. The service provides functionality for both single and multiple applicant journeys.
 
-The Frontend Application delegates a number of backend logic to the underlying services, including Persistence, Business and Submit services.
+The Frontend Application delegates a number of backend logic to the underlying services, including Orchestrator, Business and Submit services.
```
Comment (douglasrice) [inline README.md]: This paragraph doesn't really make sense as the architecture has completely changed.
I think it should say it uses Orchestrator to route specific requests to the underlying services such as business service and submit service

File: test/service-stubs/orchestrator.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const router = require('express').Router();
 const logger = require('app/components/logger')('Init');
 const app = express();
 const orchestratorServicePort = config.services.orchestrator.port;
-const orchestratorServicePath = config.services.orchestrator.path;
+const orchestratorServicePath = config.services.orchestrator.paths.submissions;
```
Comment (douglasrice) [inline test/service-stubs/orchestrator.js]: why has this changed when the ticket is specifically for removing persistence service information/code

File: README.md (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply README.md]: Done

File: test/service-stubs/orchestrator.js (hunk 2 above)
Comment (LucaDelBuonoHMCTS) [reply test/service-stubs/orchestrator.js]: Because it was wrong :)

Comment (LucaDelBuonoHMCTS) [reply test/service-stubs/orchestrator.js]:
```
  orchestrator:
    url: 'http://localhost:8888'
    paths:
      forms: '/forms/case/{ccdCaseId}'
      create: '/forms/newcase/'
      submissions: '/forms/{ccdCaseId}/submissions'
      payments: '/forms/{ccdCaseId}/payments'
      payment_updates: '/payment-updates'
      payment_submissions: '/forms/{ccdCaseId}/payment-submissions'
      fees: '/forms/{ccdCaseId}/fees'
      validations: '/forms/{ccdCaseId}/validations'
      applications: '/forms/cases'
      declarationStatuses: '/invites/{ccdCaseId}'
```

**Pull request merged**